projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -226,7 +226,12 @@ public function add_upgrades_menu( $plan = null ) {
 		}
 		parent::add_upgrades_menu( $plan );
 
-		add_submenu_page( 'paid-upgrades.php', __( 'Domains', 'jetpack' ), __( 'Domains', 'jetpack' ), 'manage_options', 'https://wordpress.com/domains/manage/' . $this->domain, null, 10 );
+		$last_upgrade_submenu_position = $this->get_submenu_item_count( 'paid-upgrades.php' );
+
+		add_submenu_page( 'paid-upgrades.php', __( 'Domains', 'jetpack' ), __( 'Domains', 'jetpack' ), 'manage_options', 'https://wordpress.com/domains/manage/' . $this->domain, null, $last_upgrade_submenu_position - 1 );
+		if ( apply_filters( 'jetpack_show_wpcom_upgrades_email_menu', false ) ) {
```

Comment:
You'll want to add a docblock to that filter, so the Jetpack Codex can pick up the information when the hooks are parsed to create documentation. 

Here is an example:
https://github.com/Automattic/jetpack/blob/d3f3117217ca825f63eebbf7784ad9ef7140bec5/projects/plugins/jetpack/modules/masterbar.php#L30-L39

Reply:
Done!